src/FirePokemon.java

```diff
@@ -0,0 +1,113 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class FirePokemon extends Pokemon {
+
+    String type = "fire";
```

Comment:
Waarom is dit niet `private static final`? Nu zou iemand kunnen zeggen `charmander.type="water"`